src/main/kotlin/com/aemtools/completion/htl/completionprovider/FileVariablesResolver.kt

```diff
@@ -127,11 +128,9 @@ object FileVariablesResolver {
                     startsWith(DATA_SLY_LIST) -> {
                         val (itemName, itemListName) = extractItemAndItemListNames(this)
 
-                        val itemClass = resolveClassForDataSlyList(it)
-
                         val tag = it.findParentByType(XmlTag::class.java) ?: return result
 
-                        if (position.within(tag)) {
+                        if (position.isWithin(tag)) {
```

Comment:
You have automatically refactored a lot of documentation with this renaming. 
And if this project mostly an open source project where then we need to define requirements for documentation.
Because as a newcomer I can't understand nothing here